cypress/e2e/tests/pages/charts/monitoring-istio.spec.ts

```diff
@@ -186,4 +189,42 @@ describe('Charts', { tags: ['@charts', '@adminUser'] }, () => {
       });
     });
   });
+
+  describe('Istio', () => {
```

Comment:
If istio is now dependent on Prometheus could we add a comment here about it?

Reply:
There is a comment below:

```
  describe('Istio local provisioning', () => {
    /**
     * Istio requires Prometheus operator to be installed, see previous steps.
     */
 ```
 
 I can move it above.

Reply:
oops, sorry, didn't spot. this location is fine